R/R.sublime-syntax

```diff
@@ -187,123 +235,242 @@ contexts:
     - match: ;
       scope: punctuation.terminator.semicolon.r
 
+###[ TYPES ]###################################################################
+
   storage-types:
     - match: \b(character|complex|double|expression|integer|list|logical|numeric|single|raw)\s*(?=\()
       scope: meta.function-call.identifier.r
       captures:
         1: storage.type.r
       push: function-call-arguments
 
+###[ STRINGS ]#################################################################
+
   strings:
     - match: (R|r)("(-*)\()
       captures:
         1: storage.type.string.r
         2: punctuation.definition.string.begin.r
       push:
-        - meta_scope: string.quoted.double.r
+        - meta_include_prototype: false
+        - meta_scope: >-
+            meta.string.r
+            string.quoted.double.r
```

Comment:
I'd follow FichteFolls arguments to only break up long scope chains into multiple rows, like Bash/Zsh which would otherwise heavily exceed 100 chars. Even then `>-` is not required.

Those simple ones can keep on a single line.